src/Storage/FileStorage.php

```diff
@@ -51,8 +60,87 @@ public function flush(): void
 
             $jsonObjects = $varDumper->asJsonObjectsMap();
             $this->filesystem->write($this->path . '/' . $this->idGenerator->getId() . '.obj.json', $jsonObjects);
+
+            $this->updateManifest();
```

Comment:
I'd call it "index". It reflects the purpose better.

Reply:
Alright.